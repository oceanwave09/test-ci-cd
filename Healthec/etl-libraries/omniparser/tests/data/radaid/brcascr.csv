batch_type,First Name,Last Name,MRN/ID,SCR_DATE,SCR_NAME,SCR_CODE,SCR_RESULT
3,First26351,Last26351,26351,10-15-2023,1,77067,1
3,First26352,Last26352,26352,09-18-2014,2,77063,4
3,First26353,Last26353,26353,12-15-2001,3,76641,6