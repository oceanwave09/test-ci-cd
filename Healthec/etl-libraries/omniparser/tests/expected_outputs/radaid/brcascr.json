[
	{
		"batch_type": "3",
		"first_name": "9e7a7d800f451bef2a6bb9e2024d7569",
		"last_name": "74967d45b9d2cfa01bd08352d5ae7802",
		"mrn": "1c26ac8bfc15d08afbccac2ab1ec40fd",
		"row_id": "bc4862c3-dd0c-47e0-9816-16d956235db0",
		"scr_code": "77067",
		"scr_date": "10-15-2023",
		"scr_name": "1",
		"scr_result": "1"
	},
	{
		"batch_type": "3",
		"first_name": "8f5a96dc34bd18e95f0fae1ef8328649",
		"last_name": "137fa153590188b32101b22447adbff0",
		"mrn": "3d7c0ad3e51ca2d9fa55d2c374bcd17a",
		"row_id": "2d844d90-3241-446c-a73d-75e880c92501",
		"scr_code": "77063",
		"scr_date": "09-18-2014",
		"scr_name": "2",
		"scr_result": "4"
	},
	{
		"batch_type": "3",
		"first_name": "7e0453536700a8d9f19f49441f1a1bab",
		"last_name": "68f3b3f9d7a7f21efee846b1a19cda3b",
		"mrn": "4d94e698dee0e06c55c995dc6f59ea55",
		"row_id": "d0d4acb9-5777-43db-b77e-79fff425faee",
		"scr_code": "76641",
		"scr_date": "12-15-2001",
		"scr_name": "3",
		"scr_result": "6"
	}
]
